op-challenger/charlie

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
For each of the bash files can we add:
```suggestion
set -euo pipefail
```

Will catch a lot of common bash coding errors and ensure the script fails if a command fails rather than silently continuing.